src/pg_client.py

```diff
@@ -58,10 +70,14 @@ def get_solver_rewards(
 
         # Here, we use the convention that we run the prod query for the first connection
         # and the barn query to all other connections
+        log.info("Setting tcp_keepalives_idle to 900 for prod connection")
+        self.connections[0].execute(text("SET tcp_keepalives_idle = 900;"))
```

Comment:
I would probably introduce a constant for this value.

Reply:
Thanks, I'll add that in a follow up PR to avoid needing a new approval here.